deploy/deploy.lib

```diff
@@ -312,9 +313,8 @@ configure_compilers() {
             #update pgi modules for network installation
             PGI_DIR=$(dirname $(which makelocalrc))
             makelocalrc ${PGI_DIR} -gcc ${GCC_DIR}/bin/gcc -gpp ${GCC_DIR}/bin/g++ -g77 ${GCC_DIR}/bin/gfortran -x -net
-
             #configure pgi network license
-            template=$(find $PGI_DIR -name localrc* | tail -n 1)
+            template=$(find $PGI_DIR -name localrc* ! -iname "*.bak" | tail -n 1)
```

Comment:
That's nasty. Did this fix your bugs?

Reply:
yes! Seems like makelocalrc now save the backup of previous config which is with gcc4.8.